src/Dolo.jl

```diff
@@ -35,17 +30,26 @@ export yaml_import, eval_with, evaluate, evaluate!, model_type, name, filename,
 
 export time_iteration, value_iteration, steady_state_residuals, simulation
 
-# set up core types
-abstract AbstractModel{ID}
-abstract AbstractSymbolicModel{ID} <: AbstractModel{ID}
-abstract AbstractNumericModel{ID} <: AbstractModel{ID}
+export ModelCalibration, FlatCalibration, GroupedCalibration
 
-typealias ASM AbstractSymbolicModel
-typealias ANM AbstractNumericModel
+# set up core types
+abstract AbstractSymbolicModel{ID}
+# abstract AbstractSymbolicModel{ID} <: AbstractModel{ID}
+abstract AbstractModel{ID} <: AbstractSymbolicModel{ID}
```

Comment:
What is the difference between `AbstractModel` and `AbstractSymbolicModel` now? Is one supposed to be a numerical object? If so isn't the old set of types where we are clear about symbolic vs numeric easier to understand?